### DISCUSSION TITLE
[Azure] Recipe/instructions for installing DnsServer on an Azure VM?

### DISCUSSION THREAD

**Discussion by mwherman2000** [TechnitiumSoftware/DnsServer#620] [Q&A]
- VM sizing
-  Network/firewall configuration/considerations
- Other details/instructions
Thank you

CC: @ruskpr

**Comment by ShreyasZare** [comment]
Thanks for asking. The installation method is the same for Azure VM. If its Windows, you run the installer and if its Linux, you run the install script. The other option is to use docker image which will work on all platforms.

VM size totally depends on each use-case. It depends on the load you are expecting and you need to discover this with a bit of trial and error.

For firewall, make sure you have both the TCP and UDP port 53 open to allow DNS to function when response size is large to fit in UDP packet.
